website/projects/spoke/src/app/app.component.ts

```diff
@@ -1,17 +1,29 @@
-import { Component, ElementRef } from '@angular/core';
+import { Component, ElementRef, OnInit } from '@angular/core';
 import { GoogleAnalyticsService } from 'ngx-google-analytics';
 import { fromEvent } from 'rxjs';
 import { tap, throttleTime } from 'rxjs/operators';
-
+import { PageState } from './core/state/page.state';
+import { MatSnackBar } from '@angular/material/snack-bar';
+import { TrackingPopupComponent } from './core/components/tracking-popup/tracking-popup.component';
 
 @Component({
   selector: 'spoke-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss']
 })
-export class AppComponent {
+export class AppComponent implements OnInit {
+
+  ngOnInit(): void {
+    if (localStorage['POPUP_CLOSED'] === 'false') {
```

Comment:
* localStorage references like this should be should be replaced with pageState references (you can always get the value directly by using pageState.snapshot.xxx).
* Remove POPUP_CLOSED references, we should only be dealing with ALLOW_TELEMETRY (in the pageState and beyond)